apps/costcontrol/js/message_handler.js

```diff
@@ -323,26 +299,47 @@
     });
   }
 
+  function sendNotification(activity, title, message, notificationData) {
+    // XXX: Hack hiding the message class in the icon URL
```

Comment:
We are not using the hack in a visible way anymore, remove the comment because it's missleading.